frontend/javascripts/oxalis/view/right-border-tabs/advanced_search_popover.tsx

```diff
@@ -171,9 +181,16 @@ export default class AdvancedSearchPopover<
                       this.setState({
                         searchQuery: evt.target.value,
                         currentPosition: null,
+                        areAllMatchesSelected: false,
                       })
                     }
-                    addonAfter={`${currentPosition + 1}/${numberOfAvailableOptions}`}
+                    addonAfter={
+                      <div style={{ minWidth: 25 }}>
+                        {!areAllMatchesSelected
+                          ? `${currentPosition + 1}/${numberOfAvailableOptions}`
+                          : ""}
```

Comment:
I'd add some placeholder here
```suggestion
                          : "all"}
```